core/whistle-1.0.0/src/wh_util.erl

```diff
@@ -1126,8 +1126,9 @@ write_pid(FileName) ->
 
 -spec ensure_started(atom()) -> 'ok' | {'error', any()}.
 ensure_started(App) when is_atom(App) ->
-    case application:start(App) of
+    case application:ensure_all_started(App) of
```

Comment:
`application:ensure_all_started/1` isn't available until R16B02; meaning: it fails at runtime (or in `make xref`) on R15B03.

Code such as [this](https://github.com/2600hz/kazoo/blob/master/applications/pusher/lib/apns/src/apns.erl#L53-L62) could go in `wh_util:ensure_all_started/1`.
Note that a bunch of `deps/` also call this function.